### ISSUE TITLE
Port log detail table to PF4

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15170]
Page: Logs / detail view (click on a log entry)

The page uses an HTML table with "info-table-ct" and the "labels" are actually `<td>`s instead of `<th>`s, making this inaccessible.

We _could_ fix the table... or outright port it to PF4 (which should also fix the a11y issues). Of course, it makes sense to do the latter and port to [PF4's description list (horizontal variant)](https://www.patternfly.org/v4/components/description-list#horizontal).

---

Once this is done, we might want to consider de-emphasizing the **ALL_UPPERCASE** labels, perhaps with CSS like the following:

```scss
.log-details .pf-c-description-list__term {
  --pf-c-description-list__term--FontWeight: var(--pf-global--FontWeight--normal);
  color: var(--pf-global--Color--200);
}
```

PF4's bolded labels in the description list wasn't designed with the intent of having all description list terms to be uppercase. They're yelling.

In a previous commit a while back, I wanted to to make these lowercase (via CSS, so copy/paste would preserve the uppercase), but was told that it's important to maintain parity with journalctl's commandline. (Is it actually case-sensitive though?) So we'd have to figure out another way to de-emhpasize the characters.

Our main font, RedHatText does not support tabular numbers, so I'd be surprised if it would support a small capitals option. Therefore, I am suggesting the above snippet (or something like it) as a one-off.

**Comment by garrett** [comment]
This port to PF4 and restyling would fix #15111.

**Comment by garrett** [comment]
Additionally, I suggested using a dynamic font family in https://github.com/cockpit-project/cockpit/issues/15111#issuecomment-762248858:

> Is there a way to have the text be in a proportional font unless it has one of the following?
>
> - it's more than one line
> - has multiple spaces in a row

The way it'd work is that the code would look for something like `(\s{2,}|\n)`, as in:

```js
journalEntryItemText.match(/(\s{2,}|\n)/)
```
_(Pseudocode. The import part is the `match()`.)_

Then, if it matches, have a class (in JSX, className, of course) to use a style to set the font to monospaced. Otherwise, it should be proportional.

**Comment by marusak** [comment]
> This port to PF4 and restyling would fix #15111.

So is this just #15111 better explained and we can close that one, or is this just part of it?

**Comment by garrett** [comment]
I think so?

It's beyond simple styling now... styling a very old implementation without a port doesn't make sense. This issue pulls from that and has a plan that should _also_ happen to address that issue.

The only thing this issue is lacking is to have @AdamWill over here too. (But this mention should fix that. :wink:)

**Comment by marusak** [comment]
While on it, fix also #13957
